extensions/Scratch_craft_2/modals.js

```diff
@@ -0,0 +1,72 @@
+// Name: Modals
+// ID: htmlalert
```

Comment:
This needs to contain your username
```suggestion
// ID: scratchCraftHTMLAlert
```